### DISCUSSION TITLE
ART 1.18.0

### DISCUSSION THREAD

**Discussion by beat-buesser** [Trusted-AI/adversarial-robustness-toolbox#2449] [Announcements]
This release of ART 1.18.0 introduces Overload Attack on object detection models and provides fast accurate loss gradients in Projected Gradient Descent for all norms.

# Added

- Added Overload Attack on object detection models (#2337)
- Added support for all norms in Projected Gradient Descent attacks (#2382)
- Added support for feature scaling in inference attacks (#2384)

# Changed

- Replaced model specific estimators for Yolo and Faster-RCNN with single estimator for all object detection models in PyTorch (#2321 )

# Removed

[None] 

# Fixed

- Fixed scaling of gradients of non-L[2, infinity] norms in Projected Gradient Descent attacks (#2382)

<hr /><em>This discussion was created from the release <a href='https://github.com/Trusted-AI/adversarial-robustness-toolbox/releases/tag/1.18.0'>ARt 1.18.0</a>.</em>
